script/test

```diff
@@ -1,2 +1,11 @@
 #!/bin/bash
-bundle exec rspec $@
+
+set -e
+
+echo "Running rspec with sassc"
+SASS_IMPLEMENTATION=sassc bundle exec rspec $@
+
+if bundle exec gem list --silent --exact --installed sass-embedded; then
```

Comment:
Would just `bundle show sass-embedded` work instead?
(Not important, just curious.)